docs/UserGuide.md

```diff
@@ -584,5 +584,8 @@ Action | Format, Examples
 * **CLI**: Command Line Interface
 * **GUI**: Graphical User Interface
 * **Inventory**: List of all items stored in StoreMando
-
+* **Item**: An item is made up of an item name, location, quantity, expiry date(optional) and tags(optional). 
+* **Similar Items**: Two items are similar if they have the same spelling for name and location but differ by letter case.
```

Comment:
I think should write as Items that have the same spelling for name and location but differ by letter case.

Reply:
Fixed.